libs/brackets-server/embedded-ext/project/main.js

```diff
@@ -32,6 +33,30 @@ define(function (require, exports, module) {
         .done(function(data) {
             _typeTable = data;
         });
+    // on opening of every project
```

Comment:
I think code below is self documented and this comment can be removed.

Reply:
done